src/Hello.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Hello = function(props) {
```

Comment:
Brian, how to you mean? It might not be something I'm familiar with yet. Unless you mean the arrow syntax, in which case I have no idea but I want to find out!